### ISSUE TITLE
What is the 'real' name of it?

### ISSUE THREAD

**Issue by rwbc** [Mangar2/Quabla#1]
Hi Volker,

Nice to see you back ;-)
I see at least two different namings so far: 'Quabla and 'Qapla'?

Greetings from Guenther (RWBC)

**Comment by Mangar2** [comment]
Hi Guenther,

great to hear from you! Indeed I started to write a new engine - do not
know how far it will get currently. And yes, many misswritings - the
true name is Qapla (Klingon from Star Treck for "success").

The "good old spike" had no Bitboards. I have build a bitboard move
generator a few years ago. Now I create a new engine with all the new
abilities of C++.

How did you find out?

Regards,

Volker

Am 18.01.2021 um 13:19 schrieb rwbc:

**Comment by rwbc** [comment]
Hi again Volker,

I just found it by accident, it was reported at some chess engines site (which often just propagates clones and recycles free github programs or otherwise free available programs/compilations for other file hosters to make some click + adaware money).
As I had started again my XB/UCI chronology in April 2016 after a nearly 6 years hiatus, I take a look there from time to time,
most work is done by an automatic site checker tool though.

I know it is a very early alpha release, but as it was already reported at that 'site', I decided to add it as a new program too.

https://docs.google.com/spreadsheets/d/1GXoy3lGhdKPJbatdw2_2dhWT3TkpthPcNtHs51MQydM

I wish you good luck and fun with the new project! BTW how is Ralf doing nowadays?
Any plans to visit talkchess too sometimes?

Best wishes,
Guenther

> 
> Hi Guenther, great to hear from you! Indeed I started to write a new engine - do not know how far it will get currently. And yes, many misswritings - the true name is Qapla (Klingon from Star Treck for "success"). The "good old spike" had no Bitboards. I have build a bitboard move generator a few years ago. Now I create a new engine with all the new abilities of C++. How did you find out? Regards, Volker
